static/js/place/overview.tsx

```diff
@@ -30,24 +32,66 @@ interface OverviewPropType {
   locale: string;
 }
 
-class Overview extends React.Component<OverviewPropType> {
+/**
+ * Holds response of /api/place/ranking, used to populate a ranking table.
+ */
+interface OverviewStateType {
+  rankingData: {
+    label: string[];
+    Population: { name: Record<string, unknown>; label: string }[];
+  };
+}
+class Overview extends React.Component<OverviewPropType, OverviewStateType> {
+  constructor(props: OverviewPropType) {
+    super(props);
+    this.state = {
+      rankingData: { label: [], Population: [] },
+    };
+  }
+
   render(): JSX.Element {
     return (
       <section className="factoid col-12">
         <div className="row">
-          <div className="col-12 col-md-4">
-            <GoogleMap dcid={this.props.dcid}></GoogleMap>
-          </div>
-          <div className="col-12 col-md-8">
-            <Ranking
-              dcid={this.props.dcid}
-              locale={this.props.locale}
-            ></Ranking>
+          <div
+            className={`col-12 ${this.shouldShowRanking() ? "col-md-4" : ""}`}
+          >
+            <div className={this.shouldShowRanking() ? "map-with-margin" : ""}>
```

Comment:
do we need this extra div? can we just do `col-12 ${this.shouldShowRanking() ? "col-md-4 map-with-margin" : ""}` in line 57

Reply:
When I tried to combine map-with-margin in the same div as col-md-4, the rankings would show up under the map instead
![image](https://user-images.githubusercontent.com/4034366/219811629-a28dad6b-5627-4fc2-b7ea-72052870f2b4.png)

Reply:
Played around with the CSS more and was able to remove the extra div!